book/07-git-tools/sections/credentials.asc

```diff
@@ -19,9 +19,8 @@ Git has a few options provided in the box:
   The downside of this approach is that your passwords are stored in cleartext in a plain file in your home directory.
 * If you're using a Mac, Git comes with an "`osxkeychain`" mode, which caches credentials in the secure keychain that's attached to your system account.
   This method stores the credentials on disk, and they never expire, but they're encrypted with the same system that stores HTTPS certificates and Safari auto-fills.
-* If you're using Windows, you can install a helper called "`Git Credential Manager for Windows.`"
-  This is similar to the "`osxkeychain`" helper described above, but uses the Windows Credential Store to control sensitive information.
-  It can be found at https://github.com/Microsoft/Git-Credential-Manager-for-Windows[].
+* If you're using Windows, you can enable the *Git Credential Manager* feature when installing https://gitforwindows.org/[Git for Windows] or separately install https://github.com/GitCredentialManager/git-credential-manager/releases/latest[the latest GCM] as a standalone service.
+  This is similar to the "`osxkeychain`" helper described above, but uses the Windows Credential Store to control sensitive information. It can also serve credentials to WSL1 or WSL2. See https://github.com/GitCredentialManager/git-credential-manager#windows[GCM Install Instructions] for more information.
```

Comment:
Mind splitting this into two lines? Diffs are much easier to parse if it's one sentence per line.